shenyu-examples/shenyu-examples-dubbo/shenyu-examples-alibaba-dubbo-service-annotation/src/main/java/org/apache/shenyu/examples/alibaba/dubbo/service/annotation/impl/DubboClassMultiParamServiceImpl.java

```diff
@@ -33,7 +33,7 @@
  * The type Dubbo multi param service.
  */
 @ShenyuDubboClient("/demo")
-@Service
+@ShenyuService(value = "ShenyuExampleApp.ShenyuExampleServer.AlibabaDubboClass.MultiParam")
 public class DubboClassMultiParamServiceImpl implements DubboClassMultiParamService {
```

Comment:
```java
@ShenyuDubboClient("/demo")
@ShenyuService(value = "ShenyuExampleApp.ShenyuExampleServer.AlibabaDubboClass.MultiParam")
```
should change to :
```java
@ShenyuService("/demo")
```
the other impls also need to change.

Reply:
Done